tests/python/utils_binding.py

```diff
@@ -170,14 +168,14 @@ def initXRT(opt):
             if (ip[i].m_type != 1):
                 continue
             opt.cu_base_addr = ip[i].m_base_address
-            print("CU[%d] %s @0x%x") % (i, ctypes.cast(ip[i].m_name, ctypes.c_char_p).value, opt.cu_base_addr)
+            print("CU[%d] %s @0x%x") % (i, ctypes.cast(ip[i].m_name, ctypes.c_char_p), opt.cu_base_addr)
 
         head = wrap_get_axlf_section(blob, AXLF_SECTION_KIND.MEM_TOPOLOGY)
         topo = mem_topology.from_buffer(data, head.contents.m_sectionOffset);
         mem = (mem_data * topo.m_count).from_buffer(data, head.contents.m_sectionOffset + 8)
 
         for i in range(topo.m_count):
-            print("[%d] %s @0x%x") % (i, ctypes.cast(mem[i].m_tag, ctypes.c_char_p).value, mem[i].mem_u2.m_base_address)
+            print("[%d] %s @0x%x") % (i, ctypes.cast(mem[i].m_tag, ctypes.c_char_p), mem[i].mem_u2.m_base_address)
```

Comment:
Same ctypes.value issue here:
```
[0] c_char_p(139712418895684) @0x4000000000
[1] c_char_p(139712418895684) @0x5000000000

```
instead of
```
[0] bank0 @0x4000000000
[1] bank1 @0x5000000000

```